apps/nerves_hub_www/lib/nerves_hub_www_web.ex

```diff
@@ -77,6 +76,16 @@ defmodule NervesHubWWWWeb do
       defp live_view_error(_) do
         "An error occurred while loading the view."
       end
+
+      unquote(view_helpers())
+    end
+  end
+
+  def live_component do
+    quote do
+      use Phoenix.LiveComponent
+
+      unquote(view_helpers())
```

Comment:
Did we have to make changes here to support liveview? I thought it was already a part of this app?

Reply:
I wouldn't say we strictly need to do it right now. This was what I was referencing when making the changes: https://hexdocs.pm/phoenix_live_view/installation.html#phx-gen-live-support

Reply:
That's fine if you needed to make the change. I just was under the impression it was already in but I must have been mistaken.